lighthouse-core/test/lib/drivers/driver-test.js

```diff
@@ -80,4 +82,23 @@ describe('Browser Driver', () => {
     assert.notEqual(opts.url, req1.url, 'opts.url changed after the redirects');
     assert.equal(opts.url, req3.url, 'opts.url matches the last redirect');
   });
+
+  it('will clear origin of a page', () => {
```

Comment:
See [comment above](https://github.com/GoogleChrome/lighthouse/pull/632#discussion_r78193037)
I added this to see if my origin got calculated right